### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by nj-io**
added navbar, mobile responsivity, fixed navbar for mobile

### CODE REVIEW

File: src/js/Navbar.jsx [hunk 1]
```diff
@@ -32,12 +32,16 @@ const NavbarDesktopLinks = styled.ul`
   li a:hover:not(.active) {
     cursor: pointer;
   }
+  @media (max-width: 700px) {
```
Comment (claireinez) [inline src/js/Navbar.jsx]: Empty block: remove it

File: src/js/Learn.jsx [hunk 2]
```diff
@@ -1,10 +1,41 @@
-import React from "react";
-import Navbar from "./Navbar";
+import React from 'react';
+import Navbar from './Navbar';
 
 const Learn = () => (
   <div>
     <Navbar page="learn" />
     <h1>Learn PAGE</h1>
+    <h1>Learn PAGE</h1>
```
Comment (claireinez) [inline src/js/Learn.jsx:8]: What are these for?

File: src/js/Navbar.jsx [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
   }
-  li a {
-    font-family: "ubuntu";
+  li {
+    float: right;
+    margin: 0 auto;
+    font-family: 'ubuntu';
     display: block;
-    color: white;
+    color: #f7b733;
```
Comment (claireinez) [inline src/js/Navbar.jsx:25]: Again, any way you can avoid writing out this hex code each time?

**Review by claireinez — CHANGES_REQUESTED** [review]

File: src/js/Learn.jsx (hunk 2 above)
Comment (nj-io) [reply src/js/Learn.jsx:8]: so we can see how it scrolls

File: src/js/Navbar.jsx (hunk 3 above)
Comment (nj-io) [reply src/js/Navbar.jsx:25]: the quickest next way would be to either write a const somewhere and then export it everywhere, which seems a bit overkill considering it wont be used much more than it already is

File: src/js/Navbar.jsx (hunk 1 above)
Comment (nj-io) [reply src/js/Navbar.jsx]: done @claireinez

**Review by claireinez — APPROVED** [review]

**Pull request merged**